packages/sdk/identity/src/odis/query.ts

```diff
@@ -106,6 +106,18 @@ export function getServiceContext(contextName = 'mainnet') {
   }
 }
 
+export function signWithDEK(message: string, signer: EncryptionKeySigner) {
+  // Sign
+  const key = ec.keyFromPrivate(hexToBuffer(signer.rawKey))
+  const sig = JSON.stringify(key.sign(message).toDER())
+  // Verify
+  const dek = key.getPublic(true, 'hex')
+  const pubkey = ec.keyFromPublic(trimLeading0x(dek), 'hex')
+  const validSignature: boolean = pubkey.verify(message, JSON.parse(sig))
+  debug(`Signature is valid: ${validSignature} signed by ${dek}`)
```

Comment:
Why is the signature being verified after is was just signed? It seems like this should be part of test code, but is not an efficient use of resources in production.